util-hadoop/src/test/java/com/google/cloud/hadoop/util/AccessTokenProviderCredentialsFactoryTest.java

```diff
@@ -14,32 +14,29 @@
 
 package com.google.cloud.hadoop.util;
 
-import static com.google.cloud.hadoop.util.HadoopCredentialConfiguration.ACCESS_TOKEN_PROVIDER_IMPL_SUFFIX;
+import static com.google.cloud.hadoop.util.HadoopCredentialsConfiguration.ACCESS_TOKEN_PROVIDER_IMPL_SUFFIX;
 import static com.google.common.truth.Truth.assertThat;
 
-import com.google.api.client.auth.oauth2.Credential;
-import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
-import com.google.api.client.util.Clock;
-import com.google.cloud.hadoop.util.CredentialFromAccessTokenProviderClassFactory.GoogleCredentialWithAccessTokenProvider;
+import com.google.auth.oauth2.AccessToken;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.hadoop.util.AccessTokenProviderCredentialsFactory.AccessTokenProviderCredentials;
 import com.google.cloud.hadoop.util.testing.TestingAccessTokenProvider;
 import com.google.common.collect.ImmutableList;
 import java.io.IOException;
-import java.security.GeneralSecurityException;
+import java.sql.Date;
```

Comment:
should we rely on java.util.date instead ?

Reply:
Ooops, good catch!